src/Util/Common.php

```diff
@@ -52,15 +52,57 @@ public static function isPharFile($path)
 
 
     /**
+     * Get absolute path, do not check exsit
+     *
+     * @param string $path The path to use.
+     *
+     * @return mixed
+     */
+    public static function absolutePath($path)
+    {
+        // Don't prefix absolute paths.
+        if (substr($path, 0, 1) !== '/') {
+            $path = getcwd().'/'.$path;
+        }
+
+        // Replace backslashes with forwardslashes.
+        $path = str_replace('\\', '/', $path);
+        // Combine multiple slashes into a single slash.
+        $path = preg_replace('/\/+/', '/', $path);
+
+        $parts = [];
+
+        foreach (explode('/', $path) as $part) {
+            // Ignore parts that have no value.
+            if (empty($part) === true || $part === '.') continue;
+            if ($part !== '..') {
+                // Cool, we found a new part.
```

Comment:
It's neither cool nor do we find anything here. Perhaps just `$parts[] = $path;`